### PR TITLE
Maintain consistency between Row and BatchRow

### PR DESCRIPTION
**Pull request by RiddhiBagadiaa**
Changed the label function name and altered the return type such that features always returns a 2D array and label always returns a 1D array to make the training set value consistent always. This also allows sklearn's partial fit function to train on individual rows if required"

### CODE REVIEW

File: client/tests/serving_test.py [hunk 1]
```diff
@@ -401,14 +400,21 @@ def test_all(self):
...
+                expected_len = len(case['expected_features'])
                 for i, r in enumerate(tset):
                     actual_len += 1
-                    actual = r.features() + [r.label()]
-                    if actual in case['expected']:
-                        case['expected'].remove(actual)
+                    actual_features = r.features()
+                    actual_label = r.label()
+                    # actual = r.features() + [r.label()]
+                    if actual_features in case['expected_features']:
```
Comment (sdreyer) [inline client/tests/serving_test.py:409]: Need a way to check that the label and feature in each iteration are in the same index. otherwise there could be an incorrect pair that passes the test. For example
Expected: 
[
[feature='a', label='True']
[feature='b', label='False']
]
Actual: 
[
[feature='a', label='False']
[feature='b', label='True']
]
Would still pass

File: client/tests/serving_test.py [hunk 2]
```diff
@@ -404,11 +403,18 @@ def test_all(self):
                 expected_len = len(case['expected'])
                 for i, r in enumerate(tset):
                     actual_len += 1
-                    actual = r.features() + [r.label()]
+                    # actual_features = r.features()
```
Comment (ahmadnazeri) [inline client/tests/serving_test.py:406]: remove the commented code if not needed.

**Review by ahmadnazeri — DISMISSED** [review]
LGTM; just remove the commented code that's not needed.

**Review by ahmadnazeri — APPROVED** [review]
